src/gm-apclientpp.cpp

```diff
@@ -41,6 +41,9 @@ static std::string script; // buffer for script result
 static APClient::Version client_version;
 static int items_handling = 0;
 static bool items_handling_changed = false;
+static std::list<std::string> bounce_games = {};
+static std::list<int> bounce_slots = {};
+static std::list<std::string> bounce_tags = {};
```

Comment:
The assignments here are not required.
```suggestion
static std::list<std::string> bounce_games;
static std::list<int> bounce_slots;
static std::list<std::string> bounce_tags;
```